packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1169,20 +1278,21 @@ export class GarbageCollector implements IGarbageCollector {
     }
 
     /**
-     * Called when a node is used. If the node is inactive, queue up an event that will be logged next time GC runs.
+     * Called when an inactive node is used after. Queue up an event that will be logged next time GC runs.
      */
     private inactiveNodeUsed(
         usageType: "Changed" | "Loaded" | "Revived",
         nodeId: string,
-        unreferencedState: UnreferencedStateTracker,
+        nodeStateTracker: UnreferencedStateTracker,
         fromNodeId?: string,
         packagePath?: readonly string[],
         currentReferenceTimestampMs = this.runtime.getCurrentReferenceTimestampMs(),
         requestHeaders?: IRequestHeader,
     ) {
         // If there is no reference timestamp to work with, no ops have been processed after creation. If so, skip
         // logging as nothing interesting would have happened worth logging.
-        if (currentReferenceTimestampMs === undefined) {
+        // If the node is active, skip logging.
+        if (currentReferenceTimestampMs === undefined || nodeStateTracker.state === UnreferencedState.Active) {
```

Comment:
Why wasn't this check already in place?

Reply:
The caller was checking this which is still true. This is just an additional check.